client/extensions/woocommerce/components/form-location-select/countries.js

```diff
@@ -98,6 +98,22 @@ class FormCountrySelectFromApi extends Component {
 	}
 }
 
+// TODO Move this to a proper selector (with tests)
+// https://github.com/Automattic/wp-calypso/pull/24571#discussion_r185268996
+const getContinentsWithCountries = ( state, continents, siteId ) => {
+	const locationsList = [];
+	continents.forEach( continent => {
```

Comment:
this could be a `map` instead a `forEach` with pushes

Reply:
Thanks! In this PR I'm just going to move it out of `connect`, but I plan on making this a proper selector + add some tests in a follow-up, so I can do some code cleanup there.